packages/types/src/interfaces/runtime/definitions.ts

```diff
@@ -43,6 +43,7 @@ export default {
       digestLevels: 'u32'
     },
     ConsensusEngineId: 'GenericConsensusEngineId',
+    CodecHash: 'H256',
     Digest: {
```

Comment:
@jacogr , is this the part of the code that defaults the hasher to H256 in case no custom hasher is specified?

Reply:
If there is no override, it will use that type.